src/Appwrite/Utopia/Response.php

```diff
@@ -445,6 +445,64 @@ public function output(Document $document, string $model): array
         return $this->payload;
     }
 
+
+    /**
+     * Output response
+     *
+     * Generate HTTP response output including the response header (+cookies) and body and prints them.
+     *
+     * @param string $body
+     *
+     * @return void
+     */
+    public function send(string $body = ''): void
```

Comment:
We don't need to replicate all the logic here, we can rely on the parent class to keep all the basic stuff.